fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/SurveyExperimentIntegrationTest.kt

```diff
@@ -37,6 +37,13 @@ class SurveyExperimentIntegrationTest {
 
     @After
     fun tearDown() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openExperimentsMenu {
+            verifyExperimentExists(experimentName)
+        }
+
         TestHelper.appContext.settings().showSecretDebugMenuThisSession = false
```

Comment:
Does this need to be in a `tearDown`? We don't usually put tests i.e, in the case here `verifyExperimentExists` in a `tearDown` functions. If there's a need to reset state or clean up in-between tests I would do it here.

Reply:
Ah gotcha, so how I had it was fine (where each test did the same little check at the end)?

Reply:
Yes that'll work. It's useful to check for each test.